core/source/sav/Sav4.cpp

```diff
@@ -893,3 +893,68 @@ std::string Sav4::pouchName(Pouch pouch) const
             return "";
     }
 }
+
+const std::set<int>& Sav4::availableItems(void) const
+{
+    static std::set<int> ret;
```

Comment:
repetition can be avoided by having just one function that incorporates this body and gets start and end values

Reply:
Good point; doing that. The sets will remain static, though; I don't want to recompute them or return a copy every time